templates/taxi/car_list.html

```diff
@@ -10,8 +10,11 @@ <h1>Car list</h1>
           {{ car.model }} ({{ car.manufacturer.name }})
         </li>
       {% endfor %}
+      <br>
```

Comment:
Make sure to use 2 whitespaces indentations in your `.html` files as per the checklist requirement.